pkg/filter/valuerange.go

```diff
@@ -0,0 +1,38 @@
+package filter
```

Comment:
I would love to see this moved to the main library: `pkg/ffuf`, this way it could be used from outside of the filters as well in the future. WIth the current implementation trying to reference it from the main library would cause a cyclic dependency import.

Reply:
Did things.